wallet/src/de/schildbach/wallet/ui/CheckPinDialog.kt

```diff
@@ -104,23 +107,21 @@ open class CheckPinDialog(
     }
 
     private val binding by viewBinding(FragmentEnterPinBinding::bind)
+    protected open val sharedModel by activityViewModels<CheckPinSharedModel>()
```

Comment:
`SharedViewModel` is unnecessary with callbacks and better to be got rid of later.